kong/templates/deploy-kong.yaml

```diff
@@ -16,6 +16,12 @@ spec:
       labels:
         app: {{ .Values.Component }}
     spec:
+      {{- if (not (empty .Values.ImagePullSecrets)) }}
+      imagePullSecrets:
+        {{- range .Values.ImagePullSecrets }}
+           - name: {{ . }}
```

Comment:
Your indentation is wrong here. Have a look at the template output.

Reply:
This has been fixed